curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/import-a-default-export.english.md

```diff
@@ -23,7 +23,7 @@ In the following code, please import the default export, <code>subtract</code>,
 ```yml
 tests:
   - text: Properly imports <code>export default</code> method.
-    testString: getUserInput => assert(getUserInput('index').match(/import\s+subtract\s+from\s+"math_functions"/g), 'Properly imports <code>export default</code> method.');
+    testString: 'getUserInput => assert(getUserInput("index").match(/import\s+subtract\s+from\s+("\.\/+math_functions"|'\.\/+math_functions')+\s{0,}?;/g), "Ensure that the format is correct and you are import the <code>subtract</code> export from <code>"math_functions"</code>");'
```

Comment:
The explanation needs to be fixed, as now it doesn't make much sense due to the errors.

Reply:
I've made a change, check it

Reply:
Yeah, but I've seen a mixture of those here of FCC. Don't know what I should use. Ready to change if someone tells me so, but this should be fine too